src/styles.variables.scss

```diff
@@ -3,3 +3,4 @@
 
 $paragraph-font-size: 20px;
 $parallax-min-height: 700px !default;
+$title-font-size: 25px;
```

Comment:
Giving this font size, you could just use h5:
$h5-fontsize: 1.64rem !default; -> This is 24.6px in desktop